### PR TITLE
Fabo/2231 delegation e2e

### PR DESCRIPTION
**Pull request by faboweb**
Part of #2231 

Note: The update in the balance header has currently a not predictable behavior which we need to fix.

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/vuex/modules/wallet.js [hunk 1]
```diff
@@ -113,6 +113,7 @@ export default ({ node }) => {
 
 function subscribeToTxs(rpcClient, address, dispatch) {
   function onTx(data) {
+    console.log(data)
```
Comment (jbibla) [inline src/vuex/modules/wallet.js]: 😱

Comment (faboweb) [reply src/vuex/modules/wallet.js]: Upsi

**Comment by faboweb** [comment]
Could happen if your testnet is already running for a while. Should not happen if it's freshly started like on the CI as there are no rewards piled up.

**Comment by colw** [comment]
It was a fresh build running for a minute or so. We'll tweak these things out as we go.

**Comment by faboweb** [comment]
I could set one validator to charge 100% commission so the rewards would be 0. Would that be a better solution?

**Comment by colw** [comment]
That could work. Different validators for different test purposes.

**Pull request merged**